ContractExamples/Cargo.toml

```diff
@@ -6,8 +6,13 @@ members = [
   "soroban-examples/timelock/"
 ]
 
+[workspace.package]
+version = "22.0.0-rc.3.1"
+edition = "2021"
+
 [workspace.dependencies]
-soroban-sdk = "21.6.0"
+soroban-sdk = { version = "22.0.0-rc.3.1" }
+soroban-token-sdk = { version = "22.0.0-rc.3.1" }
```

Comment:
Why do we need this additional dep?

Reply:
For some reason, the token example is using it as a dependency. Let me remove it